pkg/arm/data/master-startup.sh

```diff
@@ -25,6 +25,14 @@ EOF
   systemctl start docker.service
 fi
 
+# write the /root/.docker/config.json file to disk, but actually need it in
+# /var/lib/origin thanks to https://github.com/kubernetes/kubernetes/issues/45487
+DPATH=/root/.docker
```

Comment:
Does this works if we directly write file only to  `/var/lib/origin/.docker` ? I think we might want to keep it in a single location only?

Reply:
I'm only creating on both because that way you can interact with docker inside the node without extra config. Also I *think* if we just write it to /.docker works for all cases but it feels dirty to me.

Reply:
lets limit to the places we use for :)

Reply:
moved into `/var/lib/origin` only